src/components/Navbar/index.jsx

```diff
@@ -21,7 +16,9 @@ export default function Navbar() {
       `}
     >
       <div className='w-[130px]'>
-        <Image src={LogoImg} alt='logo' />
+        <Link href='/'>
+          <Image src='/images/Logo.png' alt='logo' width={1920} height={1080} />
```

Comment:
Consider changing these high values

Reply:
this one too if you zoom in in the website you will see that the pixels is so bad but with this value it is cleaner with any screen size

Reply:
done right now